qt_dotgraph/src/qt_dotgraph/node_item.py

```diff
@@ -75,6 +77,17 @@ def __init__(self, highlight_level, bounding_box, label, shape, color=None, pare
 
         self.hovershape = None
 
+    def parse_shape(self, shape, bounding_box):
+        if shape in ('box', 'rect', 'rectangle'):
+            self._graphics_item = QGraphicsRectItem(bounding_box)
+        elif shape in ('ellipse', 'oval', 'circle'):
+            self._graphics_item = QGraphicsEllipseItem(bounding_box)
+        elif shape in ('box3d'):
```

Comment:
This line is a bit tricky. With a single item in the parenthesis Python doesn't interpret that as a list. So `share='box'` would match. It needs to be: `elif shape in ('box3d', ):`. I will apply the change directly.